source/tyr/tyr/helper.py

```diff
@@ -348,3 +350,21 @@ def hide_domain(email):
     if "@" not in email:
         return email
     return "{}******".format(email.split('@')[0])
+
+
+def repost_to_another_tyr(logger, url, content, instance_name):
+    content.seek(0)
```

Comment:
Why not using the following instead of passing around a `logger`?

```python
logger = get_instance_logger(instance)
```

Reply:
Because we need to pass instance instead of instance_name